src/EditorFeatures/TestUtilities/NavigateTo/AbstractNavigateToTests.cs

```diff
@@ -91,10 +109,10 @@ private void InitializeWorkspace(TestWorkspace workspace)
                 var expectedItem = expecteditems[i];
                 var actualItem = items.ElementAt(i);
                 Assert.Equal(expectedItem.Name, actualItem.Name);
-                Assert.Equal(expectedItem.MatchKind, actualItem.MatchKind);
+                Assert.True(expectedItem.PatternMatch.Kind == actualItem.PatternMatch.Kind, string.Format("pattern: {0} expected: {1} actual: {2}", expectedItem.Name, expectedItem.PatternMatch.Kind, actualItem.PatternMatch.Kind));
```

Comment:
Because there are multiple tests in one and Assert.Equal doesn't allow you to log additional information.  I changed to Assert.True so I could indicate which pattern it was failing on.